### PR TITLE
Update UI documentation link if is present

### PR DESCRIPTION
**Pull request by rubenvp8510**
Fixes https://github.com/jaegertracing/jaeger-operator/issues/1292

Signed-off-by: Ruben Vargas Palma <ruben.vp8510@gmail.com>

### CODE REVIEW

File: pkg/strategy/controller.go [hunk 1]
```diff
@@ -226,13 +226,53 @@ func disableDependenciesTab(uiOpts map[string]interface{}, storage v1.JaegerStor
...
+			convertedItems, ok := items.([]interface{})
+			if !ok {
+				return false, 0
+			}
+			for _, item := range convertedItems {
+				itemMap, ok := item.(map[string]interface{})
+				if !ok {
+					return false, 0
+				}
+				if itemLabel, ok := itemMap["label"]; ok && itemLabel == "Documentation" {
```
Comment (jpkrohling) [inline pkg/strategy/controller.go:248]: Do we set this initially, or is it only set by users? Can we somehow check whether the value was set by us? We should really avoid changing a value that was set by users.

Comment (objectiser) [reply pkg/strategy/controller.go:248]: As this issue is related to the doc URL being supplied via CLI flag, can we just provide a fix for that situation (i.e. check if the CLI flag has been defined)?

Comment (rubenvp8510) [reply pkg/strategy/controller.go:248]: @jpkrohling TBH I don't like this either, unfortunately there is not a way to know if this was set by us or, not that I'm aware of. :( 

@objectiser This code is only executed if the documentation flag is set, See: https://github.com/jaegertracing/jaeger-operator/pull/1290/files#diff-8be2eee66bfcb8efd08e4b1fe2795663a53935ff150187ad426d371534045b01R258

Comment (objectiser) [reply pkg/strategy/controller.go:248]: @jpkrohling In this situation, if the doc URL is specified via CLI, could we just take that as the true value - so not overridden by user defined value?

**Review by objectiser — APPROVED** [review]

**Comment by jpkrohling** [comment]
About the decrease in the code coverage: looks like the error conditions aren't being tested:

image

**Pull request merged**